rancher-vcluster/rancher-vcluster.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: harvesterhci.io/v1beta1
+kind: Addon
+metadata:
+  name: rancher-vcluster
+  namespace: harvester-system
+  labels:
+    addon.harvesterhci.io/experimental: "true"
+spec:
+  enabled: false
+  repo: https://charts.loft.sh
+  version: "v0.15.2"
+  chart: vcluster
+  valuesContent: |-
+    hostname: "rancher.172.19.108.3.sslip.io"
+    rancherVersion: "v2.7.4"
+    vcluster:
+      image: rancher/k3s:v1.25.10-k3s1
+    sync:
+      ingresses:
+        enabled: "true"
+    init:
+      manifestsTemplate: |-
+        apiVersion: v1
+        kind: Namespace
+        metadata:
+          name: cattle-system
+        ---
+        apiVersion: v1
+        kind: Namespace
+        metadata:
+          name: cert-manager
+          labels:
+            certmanager.k8s.io/disable-validation: "true"
+        ---
+        apiVersion: helm.cattle.io/v1
+        kind: HelmChart
+        metadata:
+          name: cert-manager
+          namespace: kube-system
+        spec:
+          targetNamespace: cert-manager
+          repo: https://charts.jetstack.io
+          chart: cert-manager
+          version: v1.5.1
+          helmVersion: v3
+          set:
+            installCRDs: "true"
+        ---
+        apiVersion: helm.cattle.io/v1
+        kind: HelmChart
+        metadata:
+          name: rancher
+          namespace: kube-system
+        spec:
+          targetNamespace: cattle-system
+          repo: https://releases.rancher.com/server-charts/stable/
+          chart: rancher
+          set:
+            ingress.tls.source: rancher
+            hostname: {{ .Values.hostname }}
+            replicas: 1
+            version: {{ .Values.rancherVersion }}
+            global.cattle.psp.enabled: "false"
+            bootstrapPassword: password1234
```

Comment:
Is it necessary to set this? When I played around with Rancher's deploy instructions I was able to discover this value by grepping the logs for Bootstrap Password.

Reply:
Well its only needed for a seamless UX, else you'd need to login to fetch or reset password.
I am trying to avoid end users from tweaking the vcluster, but i am open to ideas.